app/Nova/Resources/Integration.php

```diff
@@ -152,6 +152,7 @@ public function fields(NovaRequest $request): array
 
             HasMany::make('UiTiD v1 Consumer Credentials', 'uiTiDv1Consumers', UiTiDv1::class),
             HasMany::make('UiTiD v2 Client Credentials (Auth0)', 'auth0Clients', Auth0Client::class),
+            HasMany::make('Keycloak client Credentials', 'keycloakClients', KeycloakClient::class),
```

Comment:
This is maybe something to hide behind the feature flag?

Reply:
I will pick that up in a next pr